docs/configuration/game.md

```diff
@@ -9,6 +9,7 @@ This is the meta configuration for the game, not for a particular thing in the g
 - `zone_starting_radius`: Radius of the playable zone. The zone is an area in the map where effects are applied to players not in it
 - `zone_modifications`: This is attribute is a list of modifications to perform to the playable zone in the map. For the specifics see below ["Zone modification"](#zone-modification)
 - `loot_interval_ms`: If present, interval in milliseconds for spawning loot crates
+- `auto_aim_max_distance`: Determines the max distance to auto aim
```

Comment:
Shouldn't this be a per character config? Even more, this should be a per skill config